container-search/src/test/java/com/yahoo/search/searchers/test/QueryValidatorTestCase.java

```diff
@@ -33,12 +30,21 @@ public void testValidation() {
         new QueryValidator().search(new Query("?query=mystring:foo"), execution);
 
         try {
-            new QueryValidator().search(new Query("?query=mytensor:foo"), execution);
-            fail("Excpected validation error");
+            new QueryValidator().search(new Query("?query=sddocname%3Aproduct%20lfmModel25KeysV0%3A9%2A%20mytensor1%3A%3E0"), execution);
```

Comment:
Consider simplifying query to only contain _mytensor1_.

Reply:
done, thanks